### PR TITLE
soap: add steps to enable/disable

### PR DESCRIPTION
**Pull request by unknwon**

### CODE REVIEW

**Review by michaellzc — APPROVED** [review]
thank you!

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 1]
```diff
@@ -124,6 +124,26 @@ Run the following SQL query for security event logs:
...
+   disableSourcegraphManagementAccess: true
+   ```
+
+1. Update the GSM value of the secret `CLOUD_SITE_CONFIG` by running `mi sync cloud-site-config`.
+1. Pull down the new GSM value to the VM by running `terraform apply` or trigger an apply on the Terraform Cloud.
+1. [Reload the manage instance](https://github.com/sourcegraph/deploy-sourcegraph-managed/actions/workflows/reload_instance.yml).
+
+For v2 instances:
+
+1. TODO
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]: @michaellzc Could you help with the v2 part? 😁

Comment (michaellzc) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]: v2 doesn't have a way to disable SOAP access yet, so maybe leave it empty for now?

```suggestion
<!-- TODO(@michaellzc): update docs after it is implemented -->

Disabling SOAP is not supported on v2
```

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 2]
```diff
@@ -124,6 +124,28 @@ Run the following SQL query for security event logs:
...
+
+1. Update the GSM value of the secret `CLOUD_SITE_CONFIG` by running `mi sync cloud-site-config`.
+1. Pull down the new GSM value to the VM by running `terraform apply` or trigger an apply on the Terraform Cloud.
+1. [Reload the manage instance](https://github.com/sourcegraph/deploy-sourcegraph-managed/actions/workflows/reload_instance.yml).
+
+For v2 instances:
+
+<!-- TODO(@michaellzc): update docs after it is implemented -->
+
+Disabling SOAP is not supported on v2
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]:
```suggestion
Disabling SOAP is currently not supported on v2.
```

File: content/departments/cloud/technical-docs/oidc_site_admin.md (hunk 1 above)
Comment (unknwon) [reply content/departments/cloud/technical-docs/oidc_site_admin.md]: Sounds good, thanks!

**Pull request merged**